platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/metrics/ReconnectMetrics.java

```diff
@@ -61,6 +63,27 @@ public class ReconnectMetrics {
      */
     private final Map<NodeId, CountPerSecond> rejectionFrequency = new HashMap<>();
 
+    private static final LongAccumulator.Config SENDER_DURATION_CONFIG = new LongAccumulator.Config(
+                    RECONNECT_CATEGORY, "senderReconnectDurationMs")
+            .withInitialValue(0)
+            .withAccumulator(Long::sum)
+            .withDescription("duration of reconnect as a sender, ms");
```

Comment:
please separate (or add) the `ms` to the `.withUnit()` builder method.

Reply:
Good idea. Just added. Please have a look.